### PR TITLE
Docs: Rename unsupported class property to UNSAFE_classname in Stack component

### PR DESCRIPTION
**Pull request by janicekt**
I found unsupported the class property in the Stack component documentation...

@literat Added missing style props everywhere and standardized the usage of symbols in the `Required` column.

### CODE REVIEW

File: packages/web-react/src/components/Stack/README.md [hunk 1]
```diff
@@ -32,12 +32,12 @@ Advanced example usage:
 
 | Prop name                 | Type     | Default | Required | Description                            |
 | ------------------------- | -------- | ------- | -------- | -------------------------------------- |
-| `class`                   | `string` | `null`  | no       | Custom CSS class                       |
 | `elementType`             | `string` | `div`   | no       | Element type of the wrapper element    |
 | `hasEndDivider`           | `bool`   | `false` | no       | Render a divider after the last item   |
 | `hasIntermediateDividers` | `bool`   | `false` | no       | Render dividers between items          |
 | `hasSpacing`              | `bool`   | `false` | no       | Apply a spacing between items          |
 | `hasStartDivider`         | `bool`   | `false` | no       | Render a divider before the first item |
+| `UNSAFE_className`        | `string` | `null`  | no       | Wrapper custom classname               |
```
Comment (dlouhak) [inline packages/web-react/src/components/Stack/README.md:40]: Shouldn't be escape hatches mentioned below the table as in [Twig package](https://github.com/lmc-eu/spirit-design-system/blob/59c4fbf8d7ce658bcab7dae7d5f5d9f55f498956/packages/web-twig/src/Resources/components/Stack/README.md?plain=1#L62)?

cc @literat

Comment (literat) [reply packages/web-react/src/components/Stack/README.md:40]: This problem is in all readmes. I have file an issue for this.

**Review by literat — APPROVED** [review]

**Review by adamkudrna — APPROVED** [review]
Thanks! 🙏🏻

**Comment by literat** [comment]
@adamkudrna I have fixed this issue in all readmes of this package. Please do a quick re-review. Thanks.

**Comment by literat** [comment]
@adamkudrna I am also sorry for so many changes but I also standardized the usage of symbols in the required column instead of using `yes` and `no`. I recommend doing the review commit by commit. Also, I can revert it and standardize the opposite if symbols are not the best ;-)

File: packages/web-react/src/components/Grid/README.md [hunk 2]
```diff
@@ -17,11 +17,13 @@ Use Grid to build multiple column layouts. This Grid works on twelve column syst
 
 ## Available props
 
-| Name          | Type                          | Description                         |
-| ------------- | ----------------------------- | ----------------------------------- |
-| `cols`        | `1`, `2`, `3`, `4`, `6`, `12` | Number of columns to use            |
-| `desktop`     | `1`, `2`, `3`, `4`, `6`, `12` | Number of columns to use on desktop |
-| `tablet`      | `1`, `2`, `3`, `4`, `6`, `12` | Number of columns to use on tablet  |
-| `elementType` | HTML element                  | Element type to use for the Grid    |
+| Name               | Type                          | Description                         |
```
[the remark this answers is not in the document]
Comment (literat) [reply packages/web-react/src/components/Grid/README.md]: Fixed.

**Review by crishpeen — APPROVED** [review]

**Pull request merged**